resources/views/people/reminders/form.blade.php

```diff
@@ -50,15 +57,15 @@
                     {{ trans('people.reminders_add_recurrent') }}
 
                     <input type="number" class="form-control frequency-type" name="frequency_number"
-                           value="1"
+                           value="{{$reminder->frequency_number ?? '1'}}"
                            min="1"
                            max="115"
                            :disabled="reminders_frequency == 'once'">
 
                     <select name="frequency_type" :disabled="reminders_frequency == 'once'">
-                        <option value="week">{{ trans('people.reminders_type_week') }}</option>
-                        <option value="month">{{ trans('people.reminders_type_month') }}</option>
-                        <option value="year">{{ trans('people.reminders_type_year') }}</option>
+                        <option value="week" {{ $update_or_add == 'edit' && $reminder->frequency_type == 'week' ? 'selected : '' }}>{{ trans('people.reminders_type_week') }}</option>
```

Comment:
This leads to an error. A `'` is missing after `selected`.
```
{{ $update_or_add == 'edit' && $reminder->frequency_type == 'week' ? 'selected HERE : '' }}
```

The same error appears on the 2 lines below.